QUESTIONS.md

```diff
@@ -1,5 +1,14 @@
 # Questions about Code, Node, Express, React
 
 What questions can you come up with?
-
+- More on React
+- More CSS please!
+- More HTML!
+- What is front end?
```

Comment:
We were discussing front end v. back end and roughly front end is everything that is seen and accessed by the user - html, css, and react are front end technologies.

Reply:
That's a very good question Jessica, I've always wondered what is front and back end are. I know what they in a nut shell, but i can really put into words what they really are.🤷🏾🤷🏽‍♂️🤷🏾